funnel/views/email.py

```diff
@@ -53,11 +55,17 @@ def send_email(subject, to, content):
         for recipient in to
     ]
     body = html2text(content)
-    html = transform(content, base_url=request.url_root)
+    html = transform(content, base_url=app.config['DEFAULT_DOMAIN'])
```

Comment:
`base_url` must be a URL, not a domain. You can use `f'https://{app.config['DEFAULT_DOMAIN']}/'` but this will probably need something more robust down the road.